customer.go

```diff
@@ -56,13 +57,14 @@ type CustomerBillingConfigurationInput struct {
 	PaymentProvider    CustomerPaymentProvider `json:"payment_provider,omitempty"`
 	ProviderCustomerID string                  `json:"provider_customer_id,omitempty"`
 	Sync               bool                    `json:"sync,omitempty"`
+	SyncWithProvider   bool                    `json:"sync_with_provider,omitempty"`
 	VatRate            float32                 `json:"vat_rate,omitempty"`
 }
 
 type CustomerBillingConfiguration struct {
 	PaymentProvider    CustomerPaymentProvider `json:"payment_provider,omitempty"`
 	ProviderCustomerID string                  `json:"provider_customer_id,omitempty"`
-	Sync               bool                    `json:"sync,omitempty"`
```

Comment:
@rsempe I have removed this field since AFAIK we don't return it in API response

Reply:
Good spot - Thanks 👍